SampleContrastClient/Program.cs

```diff
@@ -137,13 +137,12 @@ private static void DownloadAgentToDesktop(TeamServerClient client)
             }
         }
 
-        // Example usage of CheckForTrace method
-        private static bool DoesTraceExistForUrl( TeamServerClient client, string applicationId, string url )
+        // Example usage of GetTracesByUUID method
+        private static bool GetTraceByUUID( TeamServerClient client, string traceUuid, string organizationId )
```

Comment:
If we're not returning the actual trace here, let's rename this method. Something `DoesTraceExist`

Reply:
You are right. I'll change it back.